apps/pod-issuer-client/app/shared/queries/issuePOD.ts

```diff
@@ -0,0 +1,31 @@
+export const issuePOD = async (
+  podType: string,
+  token: string,
+  args: IIssuePODArgs
+) => {
+  if (!process.env.NEXT_PUBLIC_POD_ISSUER_SERVER_URL)
+    throw new Error("NEXT_PUBLIC_POD_ISSUER_SERVER_URL not set");
+
+  const response = await fetch(
+    `${process.env.NEXT_PUBLIC_POD_ISSUER_SERVER_URL}/${podType}/issue`,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${token}`
+      },
+      body: JSON.stringify({ ...args })
```

Comment:
The semaphore commitment did get converted to a string in the form input field. Maybe I should make it more clear.
I'm debating on whether we should send semaphore commitment as string or bigint here.

I think string is better, because on the server side. Express BodyParser.json middleware uses JSON.parse by default. And if we send it as bigint on the client side, we will get back a number which is the wrong. Unless we write our own BodyParser middleware to handle bigint, which is more complicated.

Reply:
I know why it is confusing, the type defined below is wrong. it should be
```

export interface IIssuePODArgs {
  semaphoreCommitment: string;
}```